src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
-        iNpUt = new Scanner(System.in);
-        StATe = uI_sTaTe.INITIALISED;
-        cOnTrOL.setUI(this);
+
+    public ReturnBookUI(ReturnItemControl control) {
+        this.control = control;
+        input = new Scanner(System.in);
+        state = UiState.INITIALISED;
+        control.setUI(this);
     }
 
 
-    public void RuN() {        
-        DiSpLaYoUtPuT("Return Book Use Case UI\n");
-        
+    public void run() {
+        displayOutput("Return Book Use Case UI\n");
+
         while (true) {
-            
-            switch (StATe) {
-            
-            case INITIALISED:
-                break;
-                
-            case READY:
-                String BoOk_InPuT_StRiNg = GeTiNpUt("Scan Book (<enter> completes): ");
-                if (BoOk_InPuT_StRiNg.length() == 0) 
-                    CoNtRoL.scanningCompleted();
-                
-                else {
-                    try {
-                        long Book_Id = Long.valueOf(BoOk_InPuT_StRiNg).longValue();
-                        CoNtRoL.itemScanned(Book_Id);
+
+            switch (state) {
+
+                case INITIALISED:
+                    break;
+
+                case READY:
+                    String bookInputString = getInput("Scan Book (<enter> completes): ");
+                    if (bookInputString.length() == 0) {
+                        control.scanningCompleted();
+                    }
+                    else {
+                        try {
+                            long bookId = Long.valueOf(bookInputString).longValue();
+                            control.itemScanned(bookId);
+                        } catch (NumberFormatException e) {
+                            displayOutput("Invalid bookId");
+                        }
                     }
-                    catch (NumberFormatException e) {
-                        DiSpLaYoUtPuT("Invalid bookId");
-                    }                    
-                }
-                break;                
-                
-            case INSPECTING:
-                String AnS = GeTiNpUt("Is book damaged? (Y/N): ");
-                boolean Is_DAmAgEd = false;
-                if (AnS.toUpperCase().equals("Y"))                     
-                    Is_DAmAgEd = true;
-                
-                CoNtRoL.dischargeLoan(Is_DAmAgEd);
-            
-            case COMPLETED:
-                DiSpLaYoUtPuT("Return processing complete");
-                return;
-            
-            default:
-                DiSpLaYoUtPuT("Unhandled state");
-                throw new RuntimeException("ReturnBookUI : unhandled state :" + StATe);            
+                    break;
+
+                case INSPECTING:
+                    String answer = getInput("Is book damaged? (Y/N): ");
+                    boolean isDamaged = false;
+                    if (answer.toUpperCase().equals("Y"))
+                        isDamaged = true;
```

Comment:
#PR42 - D6 : Please add brackets to if statement as per CSG (anywhere curly brackets can be used they MUST be used)

Reply:
https://github.com/elliotmitchum/ITC205/pull/42/commits/20d8573a2348360d7921fdb921d86190bc79da3f